### PR TITLE
Add more status to situation to allow more precise event logging

### PR DESCRIPTION
**Pull request by BenjaminJ**
refactor mapping of situations to event ApiMapper(opennms-direct), SituationToEvent(opennms-kafka) 
unit test new status

ALEC-265

### CODE REVIEW

File: features/ui/src/main/java/org/opennms/alec/rest/SituationRestImpl.java [hunk 1]
```diff
@@ -120,7 +120,7 @@ public Response rejected(String situationId, String feedback) {
                     LOG.debug("Situation {} already rejected", situationId);
                     return Response.accepted(MessageFormat.format("Situation {0} already rejected", situationId)).build();
                 }
-                feedback = String.format("reject situation [%s] -- user feedback :[%s]", situationId, feedback);
+                feedback = String.format("reject situation [%s]", situationId, feedback);
```
Comment (freddy61025) [inline features/ui/src/main/java/org/opennms/alec/rest/SituationRestImpl.java]: Although extra argument won't cause error, it still better to remove it.

**Review by freddy61025 — COMMENTED** [review]
only a minor problem.

File: features/ui/src/main/java/org/opennms/alec/rest/SituationRestImpl.java (hunk 1 above)
Comment (BenjaminJ) [reply features/ui/src/main/java/org/opennms/alec/rest/SituationRestImpl.java:123]: We'll probably use this feedback later, but for now the frontend doesn't use it

File: features/ui/src/main/java/org/opennms/alec/rest/SituationRestImpl.java [hunk 2]
```diff
@@ -208,7 +208,7 @@ public Response addAlarm(AlarmSet alarm) {
                 if (oldSituation.getAlarms().equals(alarmSet)) {
                     return ALECRestUtils.noContent();
                 }
-                String feedback = String.format("add alarm [%s] to situation [%s] -- user feedback :[%s]", alarm.getAlarmIdList(), alarm.getSituationId(), alarm.getFeedback());
+                String feedback = String.format("add alarm(s) [%s] to situation [%s]", alarm.getAlarmIdList(), alarm.getSituationId(), alarm.getFeedback());
```
Comment (aramosvizcarra) [inline features/ui/src/main/java/org/opennms/alec/rest/SituationRestImpl.java]: Same here

**Review by aramosvizcarra — CHANGES_REQUESTED** [review]
just a minor change

File: features/ui/src/main/java/org/opennms/alec/rest/SituationRestImpl.java (hunk 1 above)
Comment (aramosvizcarra) [inline features/ui/src/main/java/org/opennms/alec/rest/SituationRestImpl.java]:
```suggestion
                feedback = String.format("reject situation [%s]", situationId);
```

File: features/ui/src/main/java/org/opennms/alec/rest/SituationRestImpl.java (hunk 2 above)
Comment (aramosvizcarra) [inline features/ui/src/main/java/org/opennms/alec/rest/SituationRestImpl.java]:
```suggestion
                String feedback = String.format("add alarm(s) [%s] to situation [%s]", alarm.getAlarmIdList(), alarm.getSituationId());
```

**Review by aramosvizcarra — APPROVED** [review]
LGTM

**Pull request merged**